bin/changelog.js

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env node
+'use strict';
+
+/* eslint-disable no-console */
+
+/*
+ * External dependencies
+ */
+const { groupBy } = require( 'lodash' );
+const chalk = require( 'chalk' );
+const Octokit = require( '@octokit/rest' );
+
+/*
+ * Internal dependencies
+ */
+// @ts-ignore
```

Comment:
Curious about this ts-ignore 🤔 

Do we just need to add `resolveJsonModule: true` here?

https://github.com/WordPress/gutenberg/blob/655db778d7914a889408888a80b0f3feeaaa6ca9/bin/tsconfig.json#L3-L15

Reply:
I probably should have documented the rationale. The problem is that the file exists outside the `rootDir` of `bin`. I forget what the exact error is, but it's something to this effect.

Reply:
Ah, makes sense. Thanks 👍 

Node will let us require JSON, so it's probably a good idea to add `resolveJsonModule: true` for bin scripts regardless. Just a note for some future moment 🙂